doc/FrequentTasks.md

```diff
@@ -48,11 +54,15 @@ Note:
 Note:
 - It's best to run
     bundle install --binstubs --path=vendor/bundle
+    bundle update
   in the appropriate directories to keep your main ruby install
   clean. However do not forget to clean those up before doing an
   omnibus build, as the files there (especially .so files) can cause
   strange build failures, especially during the final consistency checks.
 
+Note:
+- On the dev-vm /dotfiles dir is mounted from your host.
```

Comment:
Maybe expand that it's useful for copying files back from the dev-vm to the host.